composer.json

```diff
@@ -56,6 +56,9 @@
             "build/themes/contrib/{$name}": ["type:drupal-theme"]
         },
         "patches": {
+            "drupal/core": {
+              "https://www.drupal.org/project/drupal/issues/1091852": "https://www.drupal.org/files/issues/2019-11-19/1091852-104.patch"
```

Comment:
Why don't we use the title of the Issue ? In this case "Display Bug when using #states (Forms API) with Ajax Request"

The ID of the issue is always present in the patch name (here "**1091852**-104").
It is more easy to know what is the patched issue with title name.

Reply:
no, see PR comments of 3039

Reply:
OK. Effectively we have to know the module name to have the URL.

My bad